### ISSUE TITLE
Warning

### ISSUE THREAD

**Issue by cor-rizzle** [foosel/homeassistant-coronavirus-hessen#8]
I see this in the log

```
2021-02-05 17:09:53 WARNING (MainThread) [homeassistant.loader] You are using a custom integration coronavirus_hessen which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-02-05 17:09:53 WARNING (MainThread) [homeassistant.loader] No 'version' key in the manifest file for custom integration 'coronavirus_hessen'. This will not be allowed in a future version of Home Assistant. Please report this to the maintainer of 'coronavirus_hessen'
```

**Comment by foosel** [comment]
I'm aware of this and pretty sure what to do, I simply haven't yet found the time due to my other responsibilities.

**Comment by foosel** [comment]
Should be fixed by 66499281f746cdd62270f05de30ffdf2a30d8615
